src/test/java/net/masterthought/cucumber/generators/AbstractPageTest.java

```diff
@@ -48,8 +47,32 @@ public void generateReport_CreatesReportFile() {
         assertThat(reportFile).exists();
     }
 
+
+    @Test
+    public void generateReport_EscapesHtml() {
+        setUpWithJson(HTML_JSON);
+
+        // given
+        page = new FeatureReportPage(reportResult, configuration, features.get(0));
+
+        // when
+        page.generatePage();
+
+        // then
+        DocumentAssertion document = documentFrom(page.getWebPage());
+        assertThat(document.getFeature().getDescription(),
+                equalTo("<b>a bold statement</b>"));
+        assertThat(document.getFeature().getElements()[0].getDescription(),
+                equalTo("<b>a bold statement</b>"));
+        assertThat(document.getFeature().getElements()[0].getStepsSection().getSteps()[0].getDocString().text(),
+                equalTo("Doc string <b>a bold\nstatement</b>"));
+        assertThat(document.getFeature().getElements()[0].getStepsSection().getSteps()[1].getArgumentsTable().getBodyRow().getCellsValues()[0],
+                equalTo("<b>bold</b>"));
+    }
+
     @Test
     public void generateReport_OnInvalidPath_ThrowsException() {
+        setUpWithJson(SAMPLE_JSON);
```

Comment:
Because leaving it in the @Before method and then calling setUpWithJson(HTML_JSON); in the escapeHtml test would be deceptive.